lib/assets/javascripts/utils/links.js

```diff
@@ -1,15 +1,29 @@
 import 'number-to-text/converters/en-us';
 import { convertToText } from 'number-to-text/index';
-import { isFunction } from './isType';
+import { isFunction, isString } from './isType';
 import { enableValidations, disableValidations } from './validation';
 
+const toggleValidations = (el) => {
+  const linkVal = el.find('input[name="link_link"]').val();
+  const textVal = el.find('input[name="link_text"]').val();
+  if (isString(linkVal) && isString(textVal)) {
+    // Validations are enabled if non-empty value is found
+    if (linkVal.length > 0 || textVal.length > 0) {
+      enableValidations(el);
+    } else {
+      disableValidations(el);
+    }
+  }
+};
+
 const getLinks = elem =>
   $(elem).find('.link').map((i, el) => {
     const link = $(el);
+    // toggleValidations(link);
     const linkVal = link.find('input[name="link_link"]').val();
     const textVal = link.find('input[name="link_text"]').val();
-    if (linkVal || textVal) { // Validations are enabled if non-empty value is found
-      enableValidations(link);
+    if (linkVal.length > 0 || textVal.length > 0) {
```

Comment:
there is no need to add .length to this conditional since in the case that linkVal and textVal are empty strings, they are considered falsy in JS and therefore the body of that if will not be reached.